src/epub/content.opf

```diff
@@ -49,7 +49,7 @@
 		<meta property="se:subject">Shorts</meta>
 		<dc:description id="description">Seven science fiction short stories by author Algis Budrys.</dc:description>
 		<meta id="long-description" property="se:long-description" refines="#description">
-			&lt;p&gt;&lt;a href="https://standardebooks.org/ebooks/algis-budrys"&gt;Algis Budrys’&lt;/a&gt; science fiction writing career is long and storied. This collection of seven of his early stories published in science fiction pulp magazines is a window into his imagination and style.&lt;/p&gt;
+			&lt;p&gt;&lt;a href="https://standardebooks.org/ebooks/algis-budrys"&gt;Algis Budrys’&lt;/a&gt; science fiction writing career is long and storied. This collection of eight of his early stories published in science fiction pulp magazines is a window into his imagination and style.&lt;/p&gt;
```

Comment:
Actually, let's just remove the number entirely so this doesn't happen again.

Reply:
Done!